### PR TITLE
Add macrosPekko and macrosPekkoTests

### PR DESCRIPTION
**Pull request by kaplanbar**
Adds `macrosPekko` and `macrosPekkoTests` to be used with Pekko.

See: https://github.com/softwaremill/macwire/issues/301

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -160,6 +161,14 @@ lazy val macrosAkka = projectMatrix
   .jvmPlatform(scalaVersions = scala2)
   .jsPlatform(scalaVersions = scala2)
 
+lazy val macrosPekko = projectMatrix
```
Comment (adamw) [inline build.sbt:164]: this should be included in the root aggregate project list

**Comment by adamw** [comment]
Other than the .sbt change I assume it's a direct copy & package change :)

File: build.sbt (hunk 1 above)
Comment (kaplanbar) [reply build.sbt:164]: 🤦‍♂️ I will add it

Comment (kaplanbar) [reply build.sbt:166]: Added.

**Comment by kaplanbar** [comment]
> Other than the .sbt change I assume it's a direct copy & package change :)

Right, it is just copy paste and fixing the imports.

`import akka.*` => `import org.apache.pekko.*`

**Comment by adamw** [comment]
Thanks :)

**Pull request merged**